roadmap.md

```diff
@@ -38,18 +52,23 @@ manage multiple wallets/agents.
 ### Migrating to aries-askar
 Currently, the main bottleneck of `aries-vcx`, especially in institutional contexts is reliance on 
 IndySDK implementation of wallet (see [issue](https://github.com/hyperledger/indy-sdk/issues/2164)). 
-[Aries-askar](https://github.com/andrewwhitehead/aries-askar) is new implementation of Aries storage 
+
+There are following solutions under consideration:
+1. Use [Aries-askar](https://github.com/andrewwhitehead/aries-askar) as a new implementation for Aries storage 
```

Comment:
`aries-askar`, the phrase `Aries storage` is a little confusing (just "storage" is enough I guess)

Reply:
indeed